### DISCUSSION TITLE
Error building the android client

### DISCUSSION THREAD

**Discussion by ningo247** [anyproto/contributors#103] [Anytype apps]
Hi, 
I'm trying to build the android client app but keep getting the same error on build
could someone help me understand how to solve this?

Error
`Could not GET 'https://maven.pkg.github.com/anyproto/anytype-heart/io/anyproto/anytype-heart-android/v0.28.7/anytype-heart-android-v0.28.7.pom'. Received status code 401 from server: Unauthorized`

Thanks in advance! I love this app and want to contribute :)

**Comment by uburoiubu** [comment]
@ningo247, thanks for reporting, there might be a problem with your personal access token. did you follow [this setup step](https://github.com/anyproto/anytype-kotlin#prerequisites)?

**Comment by ningo247** [comment]
Yes but I've created the access token on GitHub but wasn't sure if the user
ID in the github.properties file should be with the pound sign (#) or
without it?

And the other thing I wasn't sure is if I should just copy paste the
setting for the amplitude as is or create a key like for GitHub?

I only copy paste this into an apikeys.properties file:

amplitude.debug="AMPLITUDE_DEBUG_KEY"
amplitude.release="AMPLITUDE_RELEASE_KEY"
sentry_dsn="SENTRY_DSN_KEY"

Thank you again for you help!
Erez

**Reply by ygormutti** [reply]
I had some exceptions related to Sentry DSN having an invalid scheme (null). After that I changed all the three API keys to `""` (empty string) and it may have solved the issue

Not sure that did the trick, because I've modified other things in my environment, but I can confirm the app is starting and working with empty strings for API keys
